src/components/Footer.js

```diff
@@ -1,17 +1,18 @@
-import data from "@/config/app.json";
+import data from '@/config/app.json';
+import Link from 'next/link';
 
 export default function Footer() {
+  const githubUrl = "https://github.com/EddieHubCommunity/CreatorsRegistry";
+
   return (
-    <footer>
-      <div className="mx-auto max-w-3xl px-4 sm:px-6 lg:max-w-7xl lg:px-8">
-        <div className="border-t border-gray-200 py-8 text-center text-sm text-gray-500 sm:text-left">
-          <span className="block sm:inline">
-            &copy; {new Date().getFullYear()} EddieHub
-          </span>{" "}
-          <span className="block sm:inline">
-            All rights reserved | v{data.version}
-          </span>
+    <footer className="border-t border-gray-200 py-8">
+      <div className="mx-auto max-w-3xl px-4 sm:px-6 lg:max-w-7xl lg:px-8 flex justify-between items-center">
+        <div className="text-sm text-gray-500">
+          &copy; {new Date().getFullYear()} EddieHub All rights reserved | v{data.version}
```

Comment:
Formatting here is incorret

```suggestion
          &copy; {new Date().getFullYear()} EddieHub All rights reserved | v
          {data.version}
```